source/reference/kubectl-minio-plugin.rst

```diff
@@ -89,6 +89,71 @@ Subcommands
 - :mc:`~kubectl minio delete`
 - :mc:`~kubectl minio version`
 
+Environment Variables
+---------------------
+
+The :ref:`MinIO Operator <minio-operator-installation>` uses the following environment variables during startup to set configuration settings.
```

Comment:
I assume these have to be set on the host where `kubectl init` runs, not on like the K8s hosts.

Reply:
Added a note to that effect.